pom.xml

```diff
@@ -181,6 +198,7 @@
 					</sourceReferences>
 					<timestampProvider>jgit</timestampProvider>
 					<jgit.ignore>pom.xml</jgit.ignore>
+					<jgit.dirtyWorkingTree>warning</jgit.dirtyWorkingTree>
```

Comment:
Please leave it to default. We want build to fail if working tree is dirty, as otherwise it cause issue that what we've built may not be what we expect from the source.